src/com/namelessmc/java_api/NamelessAPI.java

```diff
@@ -177,13 +163,14 @@ public Website getWebsite() throws NamelessException {
 					uuid = Optional.of(NamelessAPI.websiteUuidToJavaUuid(uuidString));
 				}
 			} else {
-				uuid = Optional.empty();
+				uuid = null;
```

Comment:
It should be Optional.empty() here, because it is known that the user has no UUID. Null would mean the UUID is unknown and the API would do an additional API request to get the UUID next time it is needed.